#! /bin/sh /usr/share/dpatch/dpatch-run
## 10_pmap.1.dpatch by  <csmall@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Cleanup pmap.1 manual page #282168

@DPATCH@
diff -urNad procps-3.2.7~/pmap.1 procps-3.2.7/pmap.1
--- procps-3.2.7~/pmap.1	2004-01-29 13:39:08.000000000 +1100
+++ procps-3.2.7/pmap.1	2006-06-26 11:48:23.000000000 +1000
@@ -10,25 +10,29 @@
 pmap \- report memory map of a process
 
 .SH SYNOPSIS
-.nf
-pmap [ -x | -d ] [ -q ] pids...
-pmap -V
-.fi
+.B pmap
+.RB [ \-x | \-d ]
+.RB [ \-q ]
+.I pid
+\& ...
+.br
+.B pmap \-V
 
 .SH DESCRIPTION
 The pmap command reports the memory map of a process or processes.
 
 .SH "GENERAL OPTIONS"
 .TS
-l l l.
--x	extended	Show the extended format.
--d	device	Show the device format.
--q	quiet	Do not display some header/footer lines.
--V	show version	Displays version of program.
+lB l l.
+\-x	extended	Show the extended format.
+\-d	device	Show the device format.
+\-q	quiet	Do not display some header/footer lines.
+\-V	show version	Displays version of program.
 .TE
 
 .SH "SEE ALSO"
-ps(1) pgrep(1)
+.BR ps(1),
+.BR pgrep(1)
 
 .SH STANDARDS
 No standards apply, but pmap looks an awful lot like a SunOS command.
